### PR TITLE
Added SeaClouds Dashboard Readme.md. Part of #164

### PR DESCRIPTION
**Pull request by adriannieto**
@andreaturli could you take a look to the Readme.md?

### CODE REVIEW

File: dashboard/README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+Running SeaClouds Dashboard only requires Java 7 (or greater) installed on the target machine. Among with the software requirements, it also needs:
+
+- SeaClouds Planner, Sla, Monitor and Deployer Components up and running on an accessible endpoint.
+- 1 open port if you want to access to the Dashboard remotely.
+- A config.yml configuration file.
+
+### Config.yml configuration file
+SeaClouds Dashboard is packaged as a Dropwizard application. It requires a configuration file to run. SeaClouds already has an example [config.yml](./config.yml)
+file that takes all the required endpoints from ENV variables (specified by ${ENV_VAR_NAME}) and uses 8000 as default port. If you want to override this endpoints please replace it on the config.yml file or
+modify the ENV variables.
```
Comment (andreaturli) [inline dashboard/README.md]: please enumerate all the env variables. Are they optional? if yes, what are the defaults?

File: dashboard/README.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+# Installing and running SeaClouds Dashboard
+The easiest way to install and run this component is explained in SeaClouds [README.md](../README.md). However, there are some
+use case that may find interesting to run SeaClouds Dashboard in standalone mode, like adding new features or running SeaClouds Dashboard
+wired with a remote SeaClouds Platform instance.
+
+## Requirements
+Running SeaClouds Dashboard only requires Java 7 (or greater) installed on the target machine. Among with the software requirements, it also needs:
+
+- SeaClouds Planner, Sla, Monitor and Deployer Components up and running on an accessible endpoint.
```
Comment (andreaturli) [inline dashboard/README.md]: add pointers to those components

File: dashboard/README.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+- 1 open port if you want to access to the Dashboard remotely.
```
Comment (andreaturli) [inline dashboard/README.md]: I think this is not clear. what do you mean by remotely? I guess one wants always to access the Dashboard

File: dashboard/README.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+This component provides an easy way to interact with SeaClouds Platform by using an Angular.js web application. 
+
+# Installing and running SeaClouds Dashboard
+The easiest way to install and run this component is explained in SeaClouds [README.md](../README.md). However, there are some
+use case that may find interesting to run SeaClouds Dashboard in standalone mode, like adding new features or running SeaClouds Dashboard
+wired with a remote SeaClouds Platform instance.
+
+## Requirements
+Running SeaClouds Dashboard only requires Java 7 (or greater) installed on the target machine. Among with the software requirements, it also needs:
```
Comment (andreaturli) [inline dashboard/README.md]: add `mvn` command used to build it

File: dashboard/README.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
+SeaClouds Dashboard
+==================
+
+This component provides an easy way to interact with SeaClouds Platform by using an Angular.js web application. 
+
+# Installing and running SeaClouds Dashboard
```
Comment (andreaturli) [inline dashboard/README.md]: Split into `installing` and `running`

File: dashboard/README.md (hunk 1 above)
Comment (adriannieto) [inline dashboard/README.md]: The ENV variables is just a way to connect with the config values, maybe is better to explain what is each config.value.

**Comment by adriannieto** [comment]
@andreaturli  I think I already addressed all your coments.

**Comment by andreaturli** [comment]
thanks @adriannieto

**Pull request merged**